firstvoices/backend/views/base_views.py

```diff
@@ -123,7 +123,9 @@ def get_detail_queryset(self):
 
     def list(self, request, *args, **kwargs):
         # permissions
-        queryset = utils.filter_by_viewable(request.user, self.get_queryset())
+        queryset = utils.filter_by_viewable(
+            request.user, self.get_queryset()
+        ).distinct()
```

Comment:
This seems like a great fix, but could you please add a test for the bug situation so we know it doesn't happen again? Thanks!

Reply:
If you're busy with the ci/cd for prod just let me know and I can handle the test for this

Reply:
i'll add a test in a separate pr